cypress/pageobject/Asset/AssetCreation.ts

```diff
@@ -62,7 +62,10 @@ export class AssetPage {
     cy.get(
       "[data-testid=asset-last-serviced-on-input] input[type='text']",
     ).click();
-    cy.get("#date-input").click().type(lastServicedOn);
+    cy.get('[data-test-id="date-input"]:visible [data-time-input="0"]')
+      .click(0, 0)
+      .type(lastServicedOn);
+    cy.get("body").click(0, 0);
```

Comment:
why are you using the hardcoded position, rather try using cy.clickAndTypeDate

Reply:
Hard coded position is only used to click outside of the popup